docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -310,6 +312,25 @@ Uninstall Alluxio as follows:
 $ helm delete alluxio
 ```
 
+#### Format Journal
+
+The helm chart contains one Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+template that can be used for formatting the Alluxio journal. The template contains one Job for each Alluxio master.
+Each Job runs `alluxio formatJournal` and formats the journal for that master.
```

Comment:
There are some duplicated sentences in `kubectl` section I wasn't able to avoid.